code/game/objects/effects/spiders.dm

```diff
@@ -29,23 +29,56 @@
 
 /obj/structure/spider/stickyweb
 	icon_state = "stickyweb1"
+	var/max_integrity = 10
+	var/mobwebbiness = 30 //prob that you get stuck in the web when moving through it
+	var/projectilewebbiness = 75 //same as above, but for projectiles.
+	var/level = 1 //used for picking sprites for the web

```

Comment:
Remove these numbers and put like "normal","heavy",etc. instead. 

spider1-1 spider2-2 vs spidernormal-1 spiderheavy-2

Reply:
green hill zone didn't die for these sins but OK FINE